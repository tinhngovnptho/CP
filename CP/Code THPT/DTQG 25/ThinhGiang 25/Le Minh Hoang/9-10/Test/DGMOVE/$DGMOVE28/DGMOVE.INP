10
4322345133
55550250
1354514034
134025403244221
4320542454201351
55531122400
14040021535542031
112335325541
1125
555552555231555551
