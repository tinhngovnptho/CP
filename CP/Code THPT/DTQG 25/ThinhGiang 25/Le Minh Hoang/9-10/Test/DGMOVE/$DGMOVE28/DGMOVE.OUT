5432234133
55555020
5135414034
513402403244221
5432054245420131
55531224001
51404002153542031
511233532541
5112
555555552315555512
